routes/cart.router.js

```diff
@@ -0,0 +1,99 @@
+const express = require("express");
+const router = express.Router();
+const { CartItem } = require("../models/cart.model");
+
+router.route("/").get(async (req, res) => {
```

Comment:
You can move route operations to the `controller` folder as you did for the `user route`. It would make code more readable and accessible